### DISCUSSION TITLE
Is it possible to get it plugin work inside SCSS @apply?

### DISCUSSION THREAD

**Discussion by its-monotype** [tailwindlabs/tailwindcss#9974] [Help]
Is it possible to use this plugin inside .scss modules cause I use tailwindcss with scss (@apply)?

**Comment by gianlucajahn** [comment]
Yes. You can use the Tailwind CSS styles in your .scss module by referencing the class names provided by Tailwind CSS. For example, you can use the p-4 class to add padding to an element:

```
.my-element {
  @extend .p-4;
}
```
This will apply the padding defined by the p-4 class to the .my-element class in your .scss module.

**Reply by its-monotype** [reply]
I meant TailwindCSS **classes sorting** when using `@apply` inside SCSS. Today I noticed that classes started to sort perfectly, but for some reason, it didn’t work for me before, maybe the old version of the plugin was installed. 
And what's the point of using `@extend` if there is `@apply`?

**Reply by its-monotype** [reply]
Oh forget it, I wrote in the wrong place and therefore you did not understand what I was writing about sorting, I apologize.
